### PR TITLE
Create models as rails engine

### PR DESCRIPTION
**Pull request by eugenk**
This branch is based on the initial commit (*not* on master). It creates a new gem which is a rails engine. Here, we can develop the models just like in a Rails app.

For migrations, run `rake db:migrate`.

For future models, the generator can be invoked from here directly, as in
```
rails g model Repository name:string slug:string description:text created_at:datetime updated_at:datetime
```

To run tests, simply run `rake`, `rake spec` or `rspec` from the root of this repository.

There is, however, no rails console or server that we can use. We can only use breakpoints in the specs for trial and error development.

### CODE REVIEW

**Comment by jelmd** [comment]
Does it make sense to add the comment from above as a README ?

**Comment by eugenk** [comment]
Good point. I will put it in a more elaborate Readme.

**Comment by phyrog** [comment]
Looks good, can you add the build status badge for Travis to the README please? Otherwise :+1:

**Comment by eugenk** [comment]
Oh, right. Also, I would like to integrate Simplecov/Coveralls and Hound right from the start.

File: spec/dummy/app/assets/javascripts/application.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// compiled file. JavaScript code in this file should be added after the last require_* statement.
```
Comment (eugenk) [inline spec/dummy/app/assets/javascripts/application.js]: Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3

File: spec/dummy/app/assets/javascripts/application.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
```
Comment (eugenk) [inline spec/dummy/app/assets/javascripts/application.js]: Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3

File: spec/dummy/app/assets/javascripts/application.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
```
Comment (eugenk) [inline spec/dummy/app/assets/javascripts/application.js]: Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3

File: spec/dummy/app/assets/javascripts/application.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
```
Comment (eugenk) [inline spec/dummy/app/assets/javascripts/application.js]: Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3

File: spec/dummy/app/assets/javascripts/application.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
```
Comment (eugenk) [inline spec/dummy/app/assets/javascripts/application.js]: Let's ignore the dummy app, as stated in https://github.com/ontohub/ontohub-models/blob/330c05fac30813a8c36987b3caddcd59e7447348/.rubocop.yml#L3

**Comment by eugenk** [comment]
@phyrog, this should be all done now.

**Comment by phyrog** [comment]
:+1:

**Comment by jelmd** [comment]
Wrt. travis: IMHO it is not clever to test on an outdated/unsupported distribution, i.e. trusty should be used instead of precise.

**Pull request merged**